### PR TITLE
Fix the navbar links redirecting problem

### LINKED ISSUE #162: When we click the nav links its not going to the relevant section
- Currently, when we click the nav links its not going to the relevant section
- Find a way to fix it

### PR DESCRIPTION
**Pull request by slayeh17**
## Purpose

The purpose of this Pull Request is to fix #162 

## Goals

When nav links are clicked it takes to you to the relevant section of the page.

## Approach

Fixed some errors in dark/index.html file where there were redundant id(s) mentioned at wrong places.

### Preview Link

https://pr-163-NEW-YOSHITHA-WEBSITE.surge.sh/

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

## Test environment
 
Ubuntu

## Learning

Review of the code

### CODE REVIEW

File: dark/index.html [hunk 1]
```diff
@@ -69,13 +69,13 @@
 
     <ul class="nav-links">
       <li>
-        <a href="#home" class="nav-items" id="home" onclick="home()">home</a>
```
Comment (YoshithaRathnayake) [inline dark/index.html:72]: Don't remove this bcs there is a onclick color given to the every link by that ID. @slayeh17

File: dark/index.html [hunk 2]
```diff
@@ -69,13 +69,13 @@
 
     <ul class="nav-links">
       <li>
-        <a href="#home" class="nav-items" id="home" onclick="home()">home</a>
+        <a href="#home" class="nav-items" onclick="home()">home</a>
```
Comment (YoshithaRathnayake) [inline dark/index.html:72]: Add it like this in every link and change it in the js file also `assets/js/app.js`
```suggestion
        <a href="#home" class="nav-items" id="home-link" onclick="home()">home</a>
```

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: dark/index.html (hunk 2 above)
Comment (slayeh17) [reply dark/index.html:72]: ok working on it

Comment (YoshithaRathnayake) [reply dark/index.html:72]: Sure

**Review by YoshithaRathnayake — DISMISSED** [review]
Change the id in the js file also `assets/js/app.js` @slayeh17

**Review by YoshithaRathnayake — APPROVED** [review]
Good Job!
Thank You @slayeh17

**Comment by slayeh17** [comment]
> Good Job! Thank You @slayeh17

Glad I could help, cheers.

**Pull request merged**